src/commands/accounts/list.ts

```diff
@@ -50,16 +48,15 @@ export const accountsList = async (
 
   const loadedAccounts = await Promise.all(
     accounts.map(async (a, index) => {
-      const ethBalance = utils.fromWei(
-        (await a.getEtherBalance()).toString(),
-        'ether'
+      const ethBalance = ethers.utils.parseEther(
```

Comment:
you can just call `BigNumber.parseEther`. trying to remove the need to call ethers directly when working with the sdk